src/runtime/pkg/govmm/qemu/qemu.go

```diff
@@ -45,6 +45,8 @@ type Machine struct {
 const (
 	// MachineTypeMicrovm is the QEMU microvm machine type for amd64
 	MachineTypeMicrovm string = "microvm"
+	// (fixed) Unix Domain Socket Path served by Intel TDX Quote Generation Service
+	qgsSocketPath string = "/var/run/tdx-qgs/qgs.socket"
```

Comment:
Is this really a fixed path? Can you point us to some documentation on the QGS side that gives us at least some hope this won't change in the future / depending on the distro packages?

Reply:
Yes. It's defined [here](https://github.com/intel/SGXDataCenterAttestationPrimitives/blob/11627935938c1f50d1084262c464c54a6e7324a0/QuoteGeneration/quote_wrapper/qgs/server_main.cpp#L40) and used https://github.com/intel/SGXDataCenterAttestationPrimitives/blob/11627935938c1f50d1084262c464c54a6e7324a0/QuoteGeneration/quote_wrapper/qgs/server_main.cpp#L195-L204. the config file only understands `port`.

Reply:
Nice, please, add that to the commit message and I'll approve it after that.
Thanks, @mythi!

Reply:
updated